string[] objects_id
---
bool success

